client/profile-wizard/steps/theme/uploader.js

```diff
@@ -68,10 +69,13 @@ class ThemeUploader extends Component {
 		const classes = classnames( 'woocommerce-theme-uploader', className, {
 			'is-uploading': isUploading,
 		} );
+		const DropZoneWrapper = isWpVersion( '5.8', '>=' )
+			? Fragment
+			: DropZoneProvider;
```

Comment:
Use a `Fragment` if php version is 5.8 or above.

Reply:
I assume this means if the WP version is 5.8 or above, right?

Does `Fragment` not work with older versions?

We do have some styling issues here while uploading.  Not entirely sure if they were introduced in this PR or pre-existing.

<img width="384" alt="Screen Shot 2021-08-17 at 2 50 47 PM" src="https://user-images.githubusercontent.com/10561050/129783903-25147276-9866-4503-a991-7d02ab7db574.png">

Reply:
Let me check, and I will add a comment for the logic. It is the `DropZoneProvider` that is depreciated in WP version 5.8 and up, I am using `Fragment` as the replacement for the time being.

Reply:
As mentioned below, the styling end up breaking because the `DropzoneProvider` has been removed which the css relied up on.